andela/Lit-Kigali1-Frontend
# #163519098 Implement forgot password page using React

#### What does this PR do?
Implement forgot password page using React

#### Description of Task to be completed?
- Write tests for redux
- Write tests for App component
- Add Button as a common component
- Add Input as a common component
- Add ForgotPasswowd component
- Add ResetPassword component
- Add project's information in `README.md`

#### How should this be manually tested?
- Clone the repository
- Run `npm run start`
- Navitage to `/forgot-password` url using any browser
- Fill and submit the form
- Check your inbox for the reset password link
- Click on the reset password link
- Fill and submit your new password

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
`#163519098`

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/17350127/55607100-8f670c80-577b-11e9-856e-1b9020bb3d95.png)

![image](https://user-images.githubusercontent.com/17350127/55607166-bde4e780-577b-11e9-872c-3b75b067a3a1.png)

![image](https://user-images.githubusercontent.com/17350127/55607323-2a5fe680-577c-11e9-8a70-a750b8ec4f45.png)

[Test coverages](https://codeclimate.com/repos/5ca1a7a152587c02850004cf/settings/test_reports/5caa0740335e6d0286002ee1)

![image](https://user-images.githubusercontent.com/17350127/55685360-68643280-5955-11e9-9d92-de2a8918fd32.png)


#### Questions:
N/A
